README.md

```diff
@@ -103,6 +103,7 @@ Add your secrets here. E.g.
 # My Project Vault File
 
 vault_default_db_password: SuperSecret
+vault_django_secret_key: als0_SEcr3t
```

Comment:
Considered adding instructions for generating a value. Say, like
```python
    chars = string.ascii_letters + string.digits
    ''.join(random.choice(chars) for x in range(64))
```
Worth it? Or just confusing for people wanting to install CommCare Sync but can't read Python?